notebooks/demo.ipynb

```diff
@@ -0,0 +1,1130 @@
+{
```

Comment:
<p>This is so cool!! Liked how the generated answers when we train the model on document (which exclude faq) is similar to the one which include. Just curious about the result "none" for the sources for the question 61. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
Also, Shall we keep only the set which exclude faq? Since, we will be using this demo for testing different models.

Reply:
> Just curious about the result "none" for the sources for the question 61.
My guess would be that it generated that using the huge training data and not on the context given to it by the approach but i'll have to look into it more.

> Shall we keep only the set which exclude faq?
splitted it into two notebooks.